app/actions/automatic_topup_action.rb

```diff
@@ -8,12 +8,28 @@ class AutomaticTopupAction < ApplicationAction
   private
 
   def perform_implementation
-    transaction = Transaction.create!(
```

Comment:
It should still raise an exception if it can't create transaction. We should only fix the code/data for the current situation and go on with this code.

Could we also could improve `AutomaticTopupsService` so that it doesn't stop if one action failed in the middle?